### PR TITLE
DlgReorderlevels

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

**Comment by stevenndungu** [comment]
This dlg is still in progress

File: instat/dlgReorderLevels.vb [hunk 1]
```diff
@@ -17,46 +17,53 @@
...
 Public Class dlgReorderLevels
     Public bFirstLoad As Boolean = True
+    Private clsDefaultFunction As New RFunction
     Private Sub dlgReorderLevels_Load(sender As Object, e As EventArgs) Handles MyBase.Load
         autoTranslate(Me)
         If bFirstLoad Then
-
             InitialiseDialog()
-            'SetDefaultSettings()
             SetDefaultSettings()
```
Comment (stevekogo) [inline instat/dlgReorderLevels.vb]: This should be implemented with the new code.

**Comment by stevekogo** [comment]
@stevenndungu There is a little bit of redoing the code in a new way then it will be ready for review and merging.

**Comment by stevekogo** [comment]
@stevenndungu What is the state of this. It seems to be working well.

**Comment by stevenndungu** [comment]
@stevekogo it is ready for review. Sorry I had not mentioned that

**Comment by stevenndungu** [comment]
@africanmathsinitiative/developers this is ready for review

File: instat/dlgReorderLevels.vb [hunk 2]
```diff
@@ -17,46 +17,64 @@
...
+        ucrReorderFactor.SetParameter(New RParameter("new_level_names", 2))
+        'ucrReorderFactor.setparameterisstring()
+
+
     End Sub
+    Private Sub SetDefaults()
+        Dim clsDefaultFunction As New RFunction
+
+        ucrSelectorFactorLevelsToReorder.Reset()
+        ucrSelectorFactorLevelsToReorder.Focus()
```
Comment (lilyclements) [inline instat/dlgReorderLevels.vb]: I don't think we need to do Focus()

**Comment by stevenndungu** [comment]
@lilyclements i think the reorder is still not implemented, I have updated the branch but still does not allow setting parameter to the reorder control.

**Comment by lilyclements** [comment]
Are you sure? I updated it on my branch Reorder Descriptives. Perhaps have a look at that to check and compare your implementation?
Reorder descriptives is an open pull request so you'll have to look at it on GitHub

**Comment by dannyparsons** [comment]
If this is a problem I can merge this now and you start on a new branch if that's easier.

**Comment by stevenndungu** [comment]
@dannyparsons Yes, that could help since I have the branch updated but still it does not recognise the new additions

**Pull request merged**